cpp/oneapi/dal/algo/jaccard/detail/vertex_similarity_ops.hpp

```diff
@@ -96,6 +96,12 @@ vertex_similarity_result vertex_similarity_ops_dispatcher<Policy, Float, Method,
     const std::int64_t max_block_size =
         get_max_block_size<Float, vertex_type<Graph>>(number_elements_in_block);
     void *result_ptr = input.get_caching_builder()(max_block_size);
+
+    if (number_elements_in_block == 0) {
+        vertex_similarity_result res(homogen_table(), homogen_table(), 0);
```

Comment:
```suggestion
        vertex_similarity_result res(table(), table(), 0);
```
You can even stick just to the table, if you do not have a requirement that the result should always contain homogen tables.

Reply:
But the better way I think is to have a default constructor for `vertex_similarity_result`